examples/formatters.py

```diff
@@ -74,6 +75,13 @@ def int2word(num, separator="-"):
         print("num out of range")
 
 
+def decorate_row_obj(row_obj: MyRowObject) -> dict:
```

Comment:
I grok that you are applying the object-oriented decorator pattern here, however in Python the term **decorator** has a very specific meaning. 

In Python, a decorator is a function which takes another function as an argument and extends the behavior of that latter function without explicitly modifying it because it returns the modified function:
```Python
def a_decorator(func: Callable) -> Callable:
    """Do something here to modify func and return the modified version"""
```

Given that your "decorator" functions are not Python decorators, I would very strongly suggest calling them something else to avoid confusing end users.

This comment applies throughout this PR.

Hmm... how about "stylist" or "style" (depending on grammatical usage) as a synonym which accurately reflects their intent?